scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1

```diff
@@ -5,6 +5,7 @@ $fireFoxPathJson = Get-Content -Raw "C:\Program Files\Mozilla Firefox\distributi
 if ($fireFoxPathJson.policies.ExtensionSettings.PSObject.Properties.Name -contains "zluribrowseragent@zluri.com") {
```

Comment:
Ignore this file
This will get automatically resolved when firefox branch is merged to main

Reply:
coo